pages/common/zoxide.md

```diff
@@ -0,0 +1,29 @@
+# zoxide
+
+> Keep track of the most frequently used directories.
+> Uses a ranking algorithm to navigate to the best match.
+> More information: <https://github.com/ajeetdsouza/zoxide>.
+
+- Go to the highest-ranked directory that contains "foo" in the name:
+
+`zoxide query {{foo}}`
+
+- Go to the highest-ranked directory that contains "foo" and then "bar":
```

Comment:
Does it have to be in that order?

Reply:
After a little experimentation it turns out yes, each of those terms must occur in the path _in that order_.